### ISSUE TITLE
Add support for new sandbox escape

### ISSUE THREAD

**Issue by professorUnknown** [Brandon-T/iOUSB#4]
It is possible by using https://siguza.github.io/psychicpaper/ the new sandbox escape to make libusb work

**Comment by Brandon-T** [comment]
Yes, it is possible. It is something I can definitely check out. I saw the post on twitter the day he posted about it and this was the first thing that came to mind. I will see if I can do it after work today and I'll let you know how it goes.

After work today:

1. Test out the sandbox escape on this repo.
2. Push a small .sh file to automate all the setup so you don't have to copy all the files and stuff for IOSUSBDevice.

**Comment by Brandon-T** [comment]
@professorIsaac  It works! Using the sandbox escape and removing the `setuid` and `setguid`, I can still access the USB without being jailbroken :D 

I will push the necessary changes shortly.

**Comment by professorUnknown** [comment]
Very nice Does Libusb Xcode project have the sandbox escape also

**Comment by professorUnknown** [comment]
I want to use it for another project it is called UTM that lets you use vms in IOS and I want to allow the virtual machines communicate with usb https://github.com/utmapp/UTM

**Comment by professorUnknown** [comment]
@Brandon-T when will you commit the necessary changes for utilizing the sandbox escape

**Comment by brandonros** [comment]
Is any of this viable in 2021 with iOS 14?
